pkg/disk/disk.go

```diff
@@ -0,0 +1,238 @@
+package disk
```

Comment:
was this file copied or moved? I think we should move files first tracking them with git and then make changes